plugins/lsp/schemastore.nix

```diff
@@ -0,0 +1,166 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+helpers.vim-plugin.mkVimPlugin config {
+  name = "schemastore";
+  originalName = "SchemaStore.nvim";
+  defaultPackage = pkgs.vimPlugins.SchemaStore-nvim;
+
+  maintainers = [ maintainers.GaetanLepage ];
+
+  extraOptions =
+    let
+      schemaOpts =
+        let
+          schemaEntry = types.submodule {
```

Comment:
Personally, I'd move `schemaEntry` to the `extraOptions` let-block (alongside `schemaOpts`). I find that more readable, rather than using doubly nested lets, but personal preference.

Reply:
Fair, done.